cs-bindgen-cli/src/generate/binding.rs

```diff
@@ -44,11 +44,13 @@ pub fn quote_raw_binding(export: &Export, dll_name: &str) -> TokenStream {
             }
         }
 
-        // TODO: Only generate a drop fn if we're binding the struct as a handle type.
-        Export::Struct(item) => class::quote_drop_fn(&item.name, dll_name),
-
-        // TODO: Generate a drop function if we're binding the enum as a handle type.
-        Export::Enum(_) => Default::default(),
+        Export::Named(export) => {
+            if export.binding_style == BindingStyle::Handle {
+                class::quote_drop_fn(&export.name, dll_name)
+            } else {
+                quote! {}
+            }
```

Comment:
You could collapse this `if`/`else` block into a single line:

```suggestion
            class::quote_drop_fn(&export.name, dll_name)
```